packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantDishes/RestaurantDish.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+    <div :class="[$style['c-restaurantCard-dish']]">
+        <p :class="[$style['c-restaurantCard-dish-description']]">
+            <span
+                data-test-id="restaurant-dish-name"
+                :class="[$style['c-restaurantCard-dish-name']]">
+                {{ name }}
+            </span>
+            <span
+                v-if="calories"
```

Comment:
```suggestion
                v-if="calories || portion"
```

Reply:
I'm unsure about this. I don't think portion makes sense without calories in this context, but I could be mistaken. Considering this is a future requirement I'd prefer to leave as is for now and fix forward once we have done more exploration around the calories requirement next year :) I'll revisit this logic then.